app/views/GuidanceView.scala.html

```diff
@@ -171,13 +156,20 @@
 
     @h2("After you have disclosed", Some("5"))
 
-    @p(Html("We will check the information you have provided and if we agree with the calculations we will send a C18 Post Clearance Demand Note in the post within 14 days."))
+    @p(Html("If we agree with the calculations you’ve provided, we’ll send a C18 Post Clearance Demand Note in the post within 14 days."))
 
-    @p(Html("The demand note will confirm what is owed to HMRC and, if you have not paid by deferment, will tell you how to pay. Payment must be made within 10 days of receiving the demand note otherwise interest will become due."))
+    @p(Html("This will confirm what you owe to HMRC and, if you have not paid by deferment, will tell you how to pay. Y ou must pay within 10 days of receiving the demand note otherwise you’ll be charged interest."))
```

Comment:
```suggestion
    @p(Html("This will confirm what you owe to HMRC and, if you have not paid by deferment, will tell you how to pay. You must pay within 10 days of receiving the demand note otherwise you’ll be charged interest."))
```

Reply:
corrected the typos!